docs/en/02_Developer_Guides/13_i18n/index.md

```diff
@@ -398,9 +398,13 @@ the browser: The current locale, and the default locale as a fallback.
 The `Requirements` class has a special method to determine these includes:
 Just point it to a directory instead of a file, and the class will figure out the includes.
 
+If using this on the frontend, it's also necessary to include the stand-alone i18n
+js file.
 
 ```php
-    Requirements::add_i18n_javascript('<my-module-dir>/javascript/lang');
+$admin = ModuleLoader::getModule('silverstripe/admin');
+Requirements::add_i18n_javascript($module->getRelativeResourcePath('client/dist/js/i18n.js'));
```

Comment:
`$admin`*

Reply:
It probably needs to be `Requirements::javascript()` as well. I'm getting this in my console now:
```
i18n.js?m=1506374375:1 Uncaught ReferenceError: webpackJsonp is not defined
    at i18n.js?m=1506374375:1
(anonymous) @ i18n.js?m=1506374375:1

```

Reply:
Fixed.